webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+
+module.exports = {
+  entry: './client/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'bundle.js'
+  },
+  rules: [
+    {
+      test: /\.m?js$/,
+      exclude: /(node_modules|bower_components)/,
+      use: {
+        loader: 'babel-loader',
+        options: {
+          presets: ['@babel/preset-env', 'react', 'es2015']

```

Comment:
Don't use the `es2015` config. Also, I've never seen anybody call it `es2015`. You should rely on the latest documentation over using anything you've seen in tutorials, etc. https://babeljs.io/docs/en/6.26.3/babel-preset-es2015